core/templates/dev/head/css/oppia.css

```diff
@@ -1027,6 +1027,17 @@ textarea {
   min-width: 126px;
 }
 
+.oppia-library-group-header-active{
```

Comment:
Not according to this page: https://github.com/oppia/oppia/pull/2545/files

Also, I noticed you're rebasing? You don't need to do that -- feel free to just submit incremental commits. They'll all just get squashed-on-merge anyway, and it's easier to see what has changed from one commit to the next in the interim.